WebViewJavascriptBridge/WKWebViewJavascriptBridge.m

```diff
@@ -158,6 +158,13 @@ - (void)webView:(WKWebView *)webView didFailProvisionalNavigation:(WKNavigation
     }
 }
 
+- (void)webViewWebContentProcessDidTerminate:(WKWebView *)webView {
```

Comment:
Should this also have `if (webView != _webView) { return; }`?

Reply:
When WKWebView content crashed, the `_webView` is no been released yet!